### ISSUE TITLE
PTZ Pro not detected

### ISSUE THREAD

**Issue by unixunion** [xMRi/PTZControl#13]
This doesnt detect my PTZ pro, and I cannot get the source to breakpoint under VC2019 to figure out why the VID is not recognizing. 

My PTZ identifies perhaps differently than yours, but this code is so hard to follow, I dont know how to get the identifier to work, so I was hoping to debug and breakpoint the logic that qualifies the devices. 

Under devices my camera lists as " Logi Group Camera", it a PTZ Pro version 1. 

device instance path USB\VID_046D&PID_0858&MI_00\7&D56C633&0&0000

**Comment by BribieUC** [comment]
We have the same problem.  Our church was given a Logitech Group camera and the software does not recognize it.  We currently have a Pro 2 camera which works fine.  We would like to add a second camera to give us more flexibility.

**Comment by xMRi** [comment]
First of all: You can set a pattern on the command line of the name of cameras to detect. (see switch -device in the docs)
User -device:"*"
The next thing is: that the camera must support a specific properties/interface. That is a special one supported only by the Logitech cameras. The camera may show up, but it might not be possible to control the camera.

The code that detects the cameras is in CPTZControlDlg::OnInitDialog, (line 533 ff)

Are you using the latest firmware?

**Comment by tomazi78** [comment]
Does it happen to you that the camera loses connection with the computer and then it is not visible in the list of devices? Restarting the device fixes the problem. Do you have something like that?

**Comment by xMRi** [comment]
Yes. In rare situations we also loose camera. We have 2. One directly connected to the laptop the second connected over an active USB 3.1 Hub (docking station).
We have to reset the hub.
Since we use a different USB connection (now a USB 3 port, before an USB 2 port) it happens but really seldom. We see the effect that the picture freezes in OSB.  We also need to restart OSB.

**Comment by BribieUC** [comment]
Thanks for your help.  Added -device:"Group" to the target line in the shortcut and it worked.  We also have issues with the second camera freezing the laptop on startup occasionally needing to reboot the system.  We appreciate your Application.

**Comment by xMRi** [comment]
Freeze the complete system, or freeze OSB or my program?
If the system freezes you may need to select a different USB Hubs.
I you use 2 cameras they should reside on different USB controllers.
